### DISCUSSION TITLE
Simulate a key press from command line

### DISCUSSION THREAD

**Discussion by Kabouik** [mpv-player/mpv#14556] [Q&A] [answered]
Is it possible to simulate a key press using a command line option? I would like to have the playlist view (F8) opened by default, and to prevent it from hiding after a short time. 

F8 shows the list in terminal, sixel, and GUI, so that's exactly what I need since I am writing a script where users would switch between these views. However I would need it to be programmatic instead of a keybind. Is there an equivalent option, or alternatively, can I make mpv believe I pressed F8 without external tools?

**Comment by guidocella** [comment, marked as answer]
`--input-commands='show-text ${playlist} 4000'`

**Comment by Kabouik** [comment]
Thank you!

Is `input-command` dependent on some specific build option? I'm using `mpv 0.37.0` on two distinct machines, one x64_86 running Guix, and one aarch64 running Droidian (Debian on an Android kernel), and only the latter has support for `input-command`. The other shows: 

```
Error parsing option input-commands (option not found)
Setting commandline option --input-commands=show-text ${playlist} 4000 failed.
Exiting... (Fatal error)
```

**Comment by guidocella** [comment]
It's from mpv 0.38.

**Comment by Kabouik** [comment]
Indeed, I mistakenly thought my Droidian machine had v0.37.0 but you're right, it's v0.38.0. In the mean time, given that showing the playlist in terminal shows broken character sequences (https://github.com/mpv-player/mpv/issues/5868) and cannot be worked around without altering `mpv` sources, I gave up on the idea to show the playlist by default, and instead just list a beautified version of the lines in the playlist file `mpv` is being playing. It lacks the dynamic indicator of the current song, but hey.

Thank you for the precious help!

**Reply by guidocella** [reply]
The playlist printing is fixed in mpv 0.38
